core/runtime/src/main/java/io/quarkiverse/operatorsdk/runtime/HelmConfiguration.java

```diff
@@ -0,0 +1,15 @@
+package io.quarkiverse.operatorsdk.runtime;
+
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.quarkus.runtime.annotations.ConfigItem;
+
+@ConfigGroup
+public class HelmConfiguration {
+
+    /**
+     * Can be used to disable helm chart generation.
+     */
+    @ConfigItem(defaultValue = "true")
```

Comment:
This should be false by default, imo. We don't want to generate Helm charts on every build for sure.

Reply:
IDK, I would rather generate everything, users still can opt out, if something is not needed, but they would be aware that it's there. Also would worth to measure the time difference. Will do some naive measurement

Reply:
This is more important in the dev mode, so at the very least, helm chart generation should be disabled in dev mode, though I still think it shouldn't be active by default, the same way we don't generate bundles by default either (which is why the support is in a different module). Since helm support is in the core module, it should be deactivated by default. Generating something all the time just to make people aware of it is a bad idea, imo. This should be addressed by documentation.

Reply:
IDK, can do it for now, but my opinion is that if there is no explicit reason why not to generate it it should be there, even for dev mode, so users can see the helm chart if they add an additional dependent resource. 

Why bundle generation is a separate module, I mean why it is not just a flag to not generate it?

Reply:
> IDK, can do it for now, but my opinion is that if there is no explicit reason why not to generate it it should be there, even for dev mode, so users can see the helm chart if they add an additional dependent resource.

The core principle is to be as efficient as possible so sure, we can always say things are "fast" enough but from my point of view, if something doesn't need to be done all the time, it shouldn't.

> 
> Why bundle generation is a separate module, I mean why it is not just a flag to not generate it?

Cleaner separation of concerns. Also allows to not bring the dependencies if not used. For that matter, we might want to extract Helm support in its own module at some later point as well.